src/NuGet.Core/NuGet.Commands/RestoreCommand/Utility/NoOpRestoreUtilities.cs

```diff
@@ -158,6 +160,34 @@ internal static bool VerifyRestoreOutput(RestoreRequest request, CacheFile cache
                 return false;
             }
 
+            if (request.ForceUpdatePackageLastAccessTime)
+            {
+                foreach (var package in cacheFile.ExpectedPackageFilePaths)
```

Comment:
Are the assets file and nuget.g.props and nuget.g.targets in this list?

Reply:
Not in my (admittedly rather simple) unit test. 

`ExpectedPackageFilePaths` is composed only in a handful of places. Which scenario can you think of where `nuget.g.props` or `nuget.g.targets` would end up in that list?

Reply:
I think it's one of those that'd be there every time if it is there. 

The answer seems to be no. Resolving.